content/posts/introducing-hippo.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Introduction to Hippo: the WebAssembly PaaS"
+description: "Hippo makes it easy to run applications and services at scale"
+date: 2021-09-20 00:00:00 +0000 UTC
+authorname: "Matthew Fisher"
+author: "@bacongobbler"
+authorlink: "https://deislabs.io/"
+image: "images/twitter-card.png"
+tags: ["hippo", "webassembly"]
+---
+
+The Deislabs team is really excited about WebAssembly. In fact, we've been
+experimenting and sharing our knowledge by developing new technologies in the
+WebAssembly space, such as [Krustlet](https://github.com/krustlet/krustlet),
+[WAGI](https://github.com/deislabs/wagi),
+[bindle](https://github.com/deislabs/bindle),
+[yo-wasm](https://github.com/deislabs/yo-wasm), and
+[wasi-experimental-http](https://github.com/deislabs/wasi-experimental-http).
+
+While experimenting with these new technologies, it became clear that we need
+to make it easy for developers to deploy and test out their new ideas.
+
+Today, we're thrilled to announce a new project.
+
+We call it Hippo, an Open Source self-hosted Platform as a Service (PaaS).
+
+![hippo demo video](videos/hippo-intro.mp4)
+
+Hippo takes a fresh spin on the PaaS ecosystem, taking advantage of the
+technology WebAssembly brings to the space.
+
+Hippo works like this: An application is packaged as a _bindle_. Bindles are
+collected together in a [bindle server](https://github.com/deislabs/bindle)
+that you can search. Hippo uses `bindle` under the hood for storing and
+organizing applications.
+
+Hippo uses a concept called "Channels" to perform automatic deployments of your
+applications. More on that later.
+
+Using the `hippo` command line interface, you can upload new releases or
+prepare a bindle for local development.
+
+Hippo provides a web interface for users to register new accounts, access their
+applications, and create new environments for testing.
+
+In the future, you can use this CLI to create applications, configure channels,
+gather logs, attach TLS certificates, and other commands you'd expect to use
+with a PaaS.
+
+Hippo makes it easy to run applications and services at scale.
+
+## Hippo's Goals
+
+We built Hippo to help with two things.
+
+First, we want to make it simple to develop applications and services. Hippo
+provides a suite of tools that make it easy to scaffold and test out new ideas.
+We also make it easier for newcomers to start writing apps. **Hippo simplifies
+the development experience**.
+
+Second, we want to make it easier for teams to manage their application's
+release life cycle. Hippo introduces a concept called "Channels". Channels
+automatically deploy the latest release based on the criteria provided. Want to
+test your idea in a staging environment? Create a new "staging" channel with a
```

Comment:
_This is not of a nit on the blog post, more of broader project terminology question. Shouldn't block this PR!_

We had avoided using the word Channels in the UI, calling things Environments instead to try and limit exposure to new terms and concepts for the Hippo user (that the PaaS interface didn't require knowledge of the inner Factory workings) - I'm wondering if we should revert those labels, as all the project intros, quickstart docs etc explain things clearly in the underlying terms. 

The obfuscation in the UI is redundant at this point, and potentially confusing?

Reply:
Channels offer a trade-off between feature availability and update churn. For example, you can expect a "nightly" channel to be unstable, but updated more often compared to an "stable" channel, which is much more stable but updates are not provided as often.

An Environment indicates where the application is running. A "production environment" indicates that the application may be running on a company's production system, whereas a "development environment" could indicate that it's running on a small PC under a desk.

Because Hippo applications are all deployed under the same "environment", a "channel" is more appropriate because we want the user to convey the application's stability rather than the underlying runtime. Of course, this assumption may change in the future, but that's the goal for right now.

We should probably fix that in the UI as we reference "Channels" everywhere else across the code base as well as the docs.